### ISSUE TITLE
Unable to install on rPi 4 running Home Assistant

### ISSUE THREAD

**Issue by backslashV** [mrlt8/docker-wyze-bridge#98]
At the docker-compose up --build step, I get

```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
docker: Error response from daemon: error while creating mount source path '/config/www/repos/docker-wyze-bridge': mkdir /config: read-only file system.
```

My docker-compose.yml file:
```
version: '2.4'
services:
    wyze-bridge:
        container_name: wyze-bridge
        restart: always
        ports:
            - 1935:1935
            - 8554:8554
            - 8888:8888
        # Uncomment to use a pre-built image:
        # image: mrlt8/wyze-bridge:latest
        # or build from source:
        build: 
            context: ./app
            dockerfile: Dockerfile.arm 
        environment:
            - WYZE_EMAIL=mywyzeemail
            - WYZE_PASSWORD=mywyzepass
            - FILTER_NAMES=Cam
            #- LAN_ONLY=True
            - DEBUG_FFMPEG=true
            - FFMPEG_CMD=-i - -vcodec copy -f rtsp rtsp://0.0.0.0:8554/
        volumes: 
            - ./tokens:/tokens/
```

How can I fix this?

**Comment by mrlt8** [comment]
You need to comment out or delete the line:
```
dockerfile: Dockerfile.arm
```
Or checkout the [wiki](https://github.com/mrlt8/docker-wyze-bridge/wiki/Home-Assistant) for the Home Assistant add-on

**Comment by backslashV** [comment]
Thank you for your response. I thought the [instructions ](https://github.com/mrlt8/docker-wyze-bridge#armraspberry-pi-support)said to uncomment that line for arm64 installation.
I just noticed the HA wiki. Is there a way to filter the cameras by name here too?

**Comment by mrlt8** [comment]
whoops! I just saw the `amd64` error and assumed you were trying to build for x64!

Another user figured out how to filter with the HA add-on: https://github.com/mrlt8/docker-wyze-bridge/discussions/68#discussioncomment-1251464

**Comment by backslashV** [comment]
Thanks. I can view the stream just fine. However, no still image isn't created under /config/www/.

**Comment by mrlt8** [comment]
hmm, are you running it as a Home Assistant add-on? The still images are only pulled from the wyze api if run as an add-on. Do you see any errors in the log?

**Comment by Taylorp913** [comment]
> Thanks. I can view the stream just fine. However, no still image isn't created under /config/www/

I also can see the rstp streams but have not been able to get the :8123/local/*.jpg to load. I just get a 404 error if I try to access it in a browser.

**Comment by mrlt8** [comment]
@Taylorp913 Are you also missing the jpgs under `/config/www/`?

**Comment by Taylorp913** [comment]
Weirdly, I just checked and there is one for my wyze-cam-outdoor from saturday. Since then I have filtered out that camera and none of the other cameras have jpgs. I am running on a pi3 and getting limited results as is, so I would not count me as a datapoint. I backordered a home assistant blue and will test out your add-on again on that system. Thanks a bunch for making those changes though!

**Comment by backslashV** [comment]
> hmm, are you running it as a Home Assistant add-on? The still images are only pulled from the wyze api if run as an add-on. Do you see any errors in the log?

Yes, I'm running the add-on. No errors.

**Comment by mrlt8** [comment]
That is strange. I'll have to look into it!

Do any of the cameras have cam plus? 

V0.5.16 should print out something like `☁️ Pulling thumbnail for..` if the camera has a thumbnail uploaded to wyze.

**Comment by backslashV** [comment]
None of the cameras have cam plus. Also, I am not seeing that line in the logs.

**Comment by backslashV** [comment]
Another issue is with one of the cameras that is on a different network. I am not able to see the stream. From the logs:

```
2021/08/30 17:42:15 [Living room cam] ☁️ WARNING: Camera is connected via "P2P mode". Stream may consume additional bandwidth!
2021/08/30 17:42:15 [Living room cam] 🎉 Starting 1080p 120kb/s Stream for WyzeCam V3 in "P2P mode" FW: 4.31.0.280 IP: 192.168.1.33 WiFi: 86%
2021/08/30 17:45:16 [Living room cam] Closing FFMPEG...
2021/08/30 17:45:17 [Living room cam] [FFMPEG] FFMPEG closed
2021/08/30 17:45:20 [Living room cam] ☁️ WARNING: Camera is connected via "P2P mode". Stream may consume additional bandwidth!
2021/08/30 17:45:20 [Living room cam] 🎉 Starting 1080p 120kb/s Stream for WyzeCam V3 in "P2P mode" FW: 4.31.0.280 IP: 192.168.1.33 WiFi: 97%
2021/08/30 17:48:15 [Living room cam] Closing FFMPEG...
2021/08/30 17:48:15 [Living room cam] [FFMPEG] FFMPEG closed
2021/08/30 17:48:19 [Living room cam] ☁️ WARNING: Camera is connected via "P2P mode". Stream may consume additional bandwidth!
2021/08/30 17:48:19 [Living room cam] 🎉 Starting 1080p 120kb/s Stream for WyzeCam V3 in "P2P mode" FW: 4.31.0.280 IP: 192.168.1.33 WiFi: 99%
2021/08/30 17:51:12 [Living room cam] Closing FFMPEG...
2021/08/30 17:51:13 [Living room cam] [FFMPEG] FFMPEG closed
2021/08/30 17:51:16 [Living room cam] ☁️ WARNING: Camera is connected via "P2P mode". Stream may consume additional bandwidth!
2021/08/30 17:51:16 [Living room cam] 🎉 Starting 1080p 120kb/s Stream for WyzeCam V3 in "P2P mode" FW: 4.31.0.280 IP: 192.168.1.33 WiFi: 96%
2021/08/30 17:51:47 I [1/1] [RTSP] [conn 172.30.32.1:44958] opened
2021/08/30 17:51:47 I [1/1] [path living-room-cam] created
2021/08/30 17:51:47 I [1/1] [RTSP] [conn 172.30.32.1:44958] ERR: no one is publishing to path 'living-room-cam'
2021/08/30 17:51:47 I [1/1] [RTSP] [conn 172.30.32.1:44958] closed
```

**Comment by mrlt8** [comment]
Could you try running with `DEBUG_FFMPEG: true` and see if gives any ffmpeg related errors?

**Comment by backslashV** [comment]
Thanks. Added it but I don't see any errors in the logs.

**Comment by backslashV** [comment]
From V0.5.18

```2021/09/03 10:34:17 [py.warnings][WARNING][Living room cam] WARNING: Received incomplete frame
2021/09/03 10:34:17 [py.warnings][WARNING][Living room cam] WARNING: Lost frame
2021/09/03 10:34:19 [py.warnings][WARNING][Living room cam] WARNING: Received incomplete frame
2021/09/03 10:34:19 [py.warnings][WARNING][Living room cam] WARNING: Lost frame
2021/09/03 10:34:20 [py.warnings][WARNING][Garage cam] WARNING: Lost frame
2021/09/03 10:34:21 [py.warnings][WARNING][Living room cam] WARNING: Received incomplete frame
2021/09/03 10:34:21 [py.warnings][WARNING][Living room cam] WARNING: Lost frame```

and no RTSP is created for this non-lan camera. Lan only is unchecked in the configs.

Another problem is that changes to the config aren't saved after pressing "save". They go back to before when I restart the add-on.

**Comment by backslashV** [comment]
Hi, the issue isn't resolved with v0.6.2

I can't see the RTSP stream. _Living room cam_ is not a LAN cam btw.
```2021/09/07 12:30:35 [Living room cam] 🎉 Starting 1080p 120kb/s Stream for WyzeCam V3 in "P2P mode" FW: 4.36.0.280 IP: 192.168.1.33 WiFi: 97%
2021/09/07 12:30:35 [Living room cam] WARNING: Received incomplete frame
2021/09/07 12:30:35 [Living room cam] WARNING: Lost frame
2021/09/07 12:33:30 [Living room cam] [FFMPEG] FFMPEG closed
2021/09/07 12:33:33 [Living room cam] ☁️ WARNING: Camera is connected via "P2P mode". Stream may consume additional bandwidth!
2021/09/07 12:33:33 [Living room cam] 🎉 Starting 1080p 120kb/s Stream for WyzeCam V3 in "P2P mode" FW: 4.36.0.280 IP: 192.168.1.33 WiFi: 100%
2021/09/07 12:33:33 [Living room cam] WARNING: Received incomplete frame
2021/09/07 12:33:33 [Living room cam] WARNING: Lost frame
2021/09/07 12:36:24 [Living room cam] [FFMPEG] FFMPEG closed
2021/09/07 12:36:27 [Living room cam] ☁️ WARNING: Camera is connected via "P2P mode". Stream may consume additional bandwidth!
2021/09/07 12:36:27 [Living room cam] 🎉 Starting 1080p 120kb/s Stream for WyzeCam V3 in "P2P mode" FW: 4.36.0.280 IP: 192.168.1.33 WiFi: 99%
2021/09/07 12:36:27 [Living room cam] WARNING: Received incomplete frame
2021/09/07 12:36:27 [Living room cam] WARNING: Lost frame
2021/09/07 12:39:18 [Living room cam] [FFMPEG] FFMPEG closed
2021/09/07 12:39:22 [Living room cam] ☁️ WARNING: Camera is connected via "P2P mode". Stream may consume additional bandwidth!
2021/09/07 12:39:22 [Living room cam] 🎉 Starting 1080p 120kb/s Stream for WyzeCam V3 in "P2P mode" FW: 4.36.0.280 IP: 192.168.1.33 WiFi: 88%
2021/09/07 12:39:22 [Living room cam] WARNING: Received incomplete frame
2021/09/07 12:39:22 [Living room cam] WARNING: Lost frame
2021/09/07 12:42:12 [Living room cam] [FFMPEG] FFMPEG closed
2021/09/07 12:42:16 [Living room cam] ☁️ WARNING: Camera is connected via "P2P mode". Stream may consume additional bandwidth!
2021/09/07 12:42:16 [Living room cam] 🎉 Starting 1080p 120kb/s Stream for WyzeCam V3 in "P2P mode" FW: 4.36.0.280 IP: 192.168.1.33 WiFi: 90%
2021/09/07 12:42:16 [Living room cam] WARNING: Received incomplete frame
2021/09/07 12:42:16 [Living room cam] WARNING: Lost frame
2021/09/07 12:45:06 [Living room cam] [FFMPEG] FFMPEG closed
2021/09/07 12:45:09 [Living room cam] ☁️ WARNING: Camera is connected via "P2P mode". Stream may consume additional bandwidth!
2021/09/07 12:45:09 [Living room cam] 🎉 Starting 1080p 120kb/s Stream for WyzeCam V3 in "P2P mode" FW: 4.36.0.280 IP: 192.168.1.33 WiFi: 98%
2021/09/07 12:45:09 [Living room cam] WARNING: Received incomplete frame
2021/09/07 12:45:09 [Living room cam] WARNING: Lost frame
```
